spec/support/feature_helpers.rb

```diff
@@ -28,6 +30,21 @@ def stub_search_and_multi_result(search, query, result)
     )
   end
 
+  def stub_find_all(klass, query, result)
+    api_class = Sequencescape::Api::V2.const_get(klass.to_s.classify)
+    allow(api_class).to receive(:find_all).with(query).and_return(result)
+  end
+
+  def stub_find_all_with_pagination(klass, query, paginate, result)
+    api_class = Sequencescape::Api::V2.const_get(klass.to_s.classify)
+
+    # Return a mock that looks like a an array of results with result-set methods
+    result.define_singleton_method(:total_count) { length } # add total_count method to array
+    result.define_singleton_method(:total_pages) { (length / 5.0).ceil } # add total_pages method to array
```

Comment:
@StephenHulme I was trying to access `total_count` from the tests, with no luck! Were you able to access `total_pages from any tests?